networks/algorand/oracle.csv

```diff
@@ -0,0 +1,3 @@
+slug,provider,chain,technology,accessPrice,queryPrice,dataTypes,decentralizationModel,economicalSecurity,economicalSecurityNote,sdk,availableApis,monitoringAndAnalytics,additionalFeatures,auditsPerformed,starred,actionButtons
+goracle-mainnet,Gora,mainnet,Decentralized Oracle,$0,$0,"[""Financial data"",""Sports stats"",""Supply chain milestones"",""Healthcare metrics""]",Decentralized,TRUE,Node runners secured by token delegation and governance on Algorand blockchain,"[""TypeScript"",""Python"",""JavaScript"",""CSS"",""Cairo""]","[""REST API""]","[""Decentralized node monitoring"",""Ecosystem rewards""]","[""Industry-specific fully customizable oracle solution"",""multiple industry templates""]",,FALSE,"[""[Docs](https://github.com/GoraNetwork)""]"
```

Comment:
Please unify the technology names everywhere. Better to use word "custom" if you are unsure of the technology used